.github/workflows/release.yml

```diff
@@ -31,7 +30,7 @@ jobs:
         if: "contains(github.event.head_commit.message, '[minor]')"
         run: echo "RELEASE=minor" >> $GITHUB_ENV
 
-      - run: yarn
+      - run: yarn build
```

Comment:
Why does it need `build`?
We run `yarn` to install the modules (basically just `release-it`) then just run `release-it` to bump and release on GitHub.

Reply:
It should not release if it cannot build the application. For example, if multiple PRs being merged lead to broken code.